cg/meta/workflow/rnafusion.py

```diff
@@ -46,7 +46,7 @@ def __init__(
         self.profile: str = config.rnafusion.profile
         self.conda_env: str = config.rnafusion.conda_env
         self.conda_binary: str = config.rnafusion.conda_binary
-        self.tower_binary_path: str = config.rnafusion.tower_binary_path
```

Comment:
Do you need to do the same for raredisease?

Reply:
that part is not implemented yet on rarediseases

Reply:
Then that makes sense!